fluidsynth/src/sfloader/fluid_defsfont.c

```diff
@@ -1817,6 +1829,61 @@ fluid_sample_in_rom(fluid_sample_t* sample)
 /*
  * fluid_sample_import_sfont
  */
+#if LIBSNDFILE_SUPPORT
+// virtual file access rountines to allow for handling
+// samples as virtual files in memory
+static sf_count_t sfvio_pos;
```

Comment:
I dont think introducing this global var here is correct. It may work because we only open one sample at a time, do all the necessary work and never come back to it again. However `_fluid_sample_t` has a member `void* userdata`, I wouldnt mind at all abusing this one for storing a `sf_count_t sfvio_pos` for that specific sample. Imagine something like
```
case SEEK_SET:
      sample->userdata = (void*) offset;
...
sfvio_tell (void* user_data)
{
   fluid_sample_t *sample = (fluid_sample_t *)user_data;
   return (sf_count_t)sample->userdata;
}
```
... not nice but more suited IMO.